tests/openvino/test_quantization.py

```diff
@@ -750,7 +750,7 @@ class OVTrainerTest(unittest.TestCase):
     @parameterized.expand(SUPPORTED_ARCHITECTURES_WITH_EXPECTED_QUANTIZED_MATMULS)
     def test_aware_training_quantization(self, model_name, expected_fake_quantize, expected_int8):
         model_id = MODEL_NAMES[model_name]
-        model = AutoModelForSequenceClassification.from_pretrained(model_id)
+        model = AutoModelForSequenceClassification.from_pretrained(model_id, attn_implementation="eager")
```

Comment:
do you know what the reason to use eager attention here? Is it impossible to quantize model with sdpa?